GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.cs

```diff
@@ -750,7 +750,7 @@ private void tsmiOpenFolder_Click(object sender, EventArgs e)
             }
             catch (Exception ex)
             {
-                MessageBox.Show(this, ex.Message);
+                MessageBox.Show(this, ex.Message, "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
```

Comment:
📝  Here and other places the heading must be localised
Please put "Error" into https://github.com/gitextensions/gitextensions/blob/master/GitUI/Strings.cs

Reply:
Fine. I noticed that never "Error" is used but always a very short custom message like "Host Fingerprint not registered" for a full message "The fingerprint of this host is not registered by PuTTY.". Should I keep this rule or is it fine to use "Error" which is redundant with the icon ?

The title is a real pain. It always redundant with the message. I'm happy the title has been remove in GTK in Linux.

Reply:
I don't think it is redundant, and quite likely it is used for accessibility reasons.
As a minimum I suggest we translate all outstanding existing headings. Later on we can look at providing better wording.